lightning/src/ln/peer_handler.rs

```diff
@@ -501,266 +531,214 @@ impl<Descriptor: SocketDescriptor, CM: Deref> PeerManager<Descriptor, CM> where
 								}
 							}
 
-							macro_rules! insert_node_id {
```

Comment:
Why remove this? We would no longer insert into the map without it, AFAICT. This breaks other code that needs the map to look up peers by id.

Reply:
None of the unit tests are failing, but I will re-add it to the graph, and then think of unit tests for this in a separate diff